common/lib/protocol-definitions/src/summary.ts

```diff
@@ -21,41 +21,88 @@ export interface ISummaryCommitter {
     date: string;
 }
 
+/**
+ *  Represents a node from the Summary Tree.
```

Comment:
I don't think this comment is accurate. I think something like this would be better:
```suggestion
 *  Type tag used to distinguish different types of nodes in a {@link ISummaryTree}.
```